docs/conf.py

```diff
@@ -270,6 +273,8 @@
     "github_repo": "astropy",
     "github_version": "main",
     "doc_path": "docs",
+    # Tell Jinja2 templates the build is running on Read the Docs
+    "READTHEDOCS": os.environ.get("READTHEDOCS", "") == "True",
```

Comment:
@humitos , can I do it like this, or does this have to be nested in an `if` strictly as stated in the email I received?

```python
import os

# Set canonical URL from the Read the Docs Domain
html_baseurl = os.environ.get("READTHEDOCS_CANONICAL_URL", "")

# Tell Jinja2 templates the build is running on Read the Docs
if os.environ.get("READTHEDOCS", "") == "True":
    html_context["READTHEDOCS"] = True
```

Reply:
Yes, you can do it like you've done it here. The snipped in the email is just an example 👍🏼